src/model/Entity.java

```diff
@@ -0,0 +1,16 @@
+package model;
+
+public abstract class Entity extends GameObject {
+
+    protected float speed;//todo

```

Comment:
this was supposed to force sub classes to have a speed value (maybe added it to the constructor or smth)